contracts/interfaces/ITwabRewards.sol

```diff
@@ -22,7 +22,6 @@ interface ITwabRewards {
      */
     struct Promotion {
```

Comment:
The promotion can be packed more tightly, which will allow larger uints.

Keep in mind `address` is 20 bytes, or 160 bits.

32 bit timestamp overflow in 100 years.  Lots of time yes, but we can remove all worries by making timestamps as large as possible.

Like so:

```
struct Promotion {
  address creator // 160 bits
  IERC20 token // 320 bits
  uint128 startTimestamp // 448 bits
  uint56 epochDuration // 504 bits
  uint8 numberOfEpochs // 512 bits
  uint256 tokensPerEpoch // 768 bits
}
```

total words: 3 (same as before)

Reply:
The order of variables matter for packing to work. As stated in this article: https://www.maxgrok.com/posts/one-hundred-and-nineth-post
Keeping this in mind, I propose this packing:
```
    struct Promotion {
        address creator; // 160 bits
        uint64 startTimestamp; // 224 bits
        uint8 numberOfEpochs; // 232 bits -> one word
        uint64 epochDuration; // 64 bits
        IERC20 token; // 224 bits -> one word
        uint256 tokensPerEpoch; // 256 bits -> one word
    }
```

This way, we still have 3 words and we don't have to cast timestamps to uint64 to interact with the `getAverageBalanceBetween` function: https://github.com/pooltogether/v4-periphery/blob/0e2de252f61725f44999016cb769fdc99539b641/contracts/TwabRewards.sol#L361-L372